jit/src/main/java/org/dcsa/conformance/standards/jit/model/JitTimestampType.java

```diff
@@ -0,0 +1,28 @@
+package org.dcsa.conformance.standards.jit.model;
+
+import lombok.Getter;
+
+@Getter
+public enum JitTimestampType {
+  ESTIMATED(JitSchema.ESTIMATED_TIMESTAMP, JitClassifierCode.EST),
+  PLANNED(JitSchema.PLANNED_TIMESTAMP, JitClassifierCode.PLN),
+  ACTUAL(JitSchema.ACTUAL_TIMESTAMP, JitClassifierCode.ACT),
+  REQUESTED(JitSchema.REQUESTED_TIMESTAMP, JitClassifierCode.REQ);
+
+  private final JitSchema jitSchema;
+  private final JitClassifierCode classifierCode;
+
+  JitTimestampType(JitSchema jitSchema, JitClassifierCode classifierCode) {
+    this.jitSchema = jitSchema;
+    this.classifierCode = classifierCode;
+  }
```

Comment:
Replace with `@RequiredArgsConstructor`?

Reply:
Ah yes, missed that. Thanks!